.github/PULL_REQUEST_GUIDELINES/rule_new_guidelines.md

```diff
@@ -0,0 +1,31 @@
+## Rule: New - Guidelines
+
+These guidelines serve as a reminder set of considerations when proposing a new rule.
+
+### Documentation and Context
+
+- [ ] Detailed description of the rule.
+- [ ] List any new fields required in ECS/data sources.
+- [ ] Link related issues or PRs.
+- [ ] Include references.
+
+### Rule Metadata Checks
+
+- [ ] `creation_date` matches the date of creation PR initially merged.
+- [ ] `min_stack_version` should support the widest stack versions.
+- [ ] `name` and `description` should be descriptive and not include typos.
+- [ ] `query` should be inclusive, not overly exclusive, considering performance for diverse environments.
+- [ ] `min_stack_comments` and `min_stack_version` should be included if the rule is only compatible starting from a specific stack version.
+- [ ] `integration` should align with the `index`. If the integration is newly introduced, ensure the manifest and schemas are updated.
+- [ ] `setup` should include the necessary steps to configure the integration.
+- [ ] `note` should include any additional information (e.g. Triage and analysis investigation guides, timeline templates).
+- [ ] `tags` should be relevant to the threat and align/added to the `EXPECTED_RULE_TAGS` in the definitions.py file.
```

Comment:
I don't recall if it's mandatory but we should map to ATT&CK always if possible to stay consistent